src/components/Statistics.tsx

```diff
@@ -92,81 +122,146 @@ const Statistics = ({ refresh }: Props) => {
     const fetchAvgBlockTimeData = async () => {
       const { data } = await client.infos.getInfosAverageBlockTimes()
       if (data && data.length > 0)
-        updateStats('avgBlockTime', data.reduce((acc: number, { value }) => acc + value, 0.0) / data.length)
+        updateStatsScalar('avgBlockTime', data.reduce((acc: number, { value }) => acc + value, 0.0) / data.length)
+    }
+
+    const fetchTxPerDayData = async () => {
+      // TODO: Replace with a real call.
```

Comment:
Is that because the endpoint is not yet ready?

Reply:
Yep